//
// This file is part of the WoW++ project.
// 
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
// GNU General Public License for more details.
// 
// You should have received a copy of the GNU General Public License
// along with this program; if not, write to the Free Software 
// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
//
// World of Warcraft, and all World of Warcraft or Warcraft art, images,
// and lore are copyrighted by Blizzard Entertainment, Inc.
// 

syntax = "proto2";
package wowpp.proto;

message QuestEntry {
	required uint32 id = 1;
	required string name = 2;
	optional uint32 method = 3 [default = 2];
	optional uint32 minlevel = 4 [default = 1];
	optional uint32 questlevel = 5;
	optional string detailstext = 6;
	optional string objectivestext = 7;
	optional string offerrewardtext = 8;
	optional string requestitemstext = 9;
	optional string endtext = 10;
}

message Quests {
	repeated QuestEntry entry = 1;
}
